spec/features/events_spec.rb

```diff
@@ -90,8 +88,30 @@
     click_button("Select recipients")
     find('a', :text => "Accepted").click
     page.should have_field('To', with: '')
+    click_button("Select recipients")
+    find('a', :text => "All").click
+    page.should have_field('To', with: 'gial.ackbar@rebel-alliance.org,luke.skywalker@rebel-alliance.org,wedge.antilles@rebel-alliance.org,evram.lajaie@rebel-alliance.org,c3po@droids.com')
+
+    click_button("Send")
+    page.should have_content "Email Delivered"
+    ActionMailer::Base.deliveries.size.should == @deliveries + 5
+  end
+
+  scenario "Cancel the email", :js => true do
+    sign_in_as_user(users(:tspmember))
+    visit events_path 
+    click_link "Death Star's destruction celebration"
+    
+    click_link "Email"
+    page.should have_content "Email the participants of Death Star's destruction celebration"
+    @deliveries = ActionMailer::Base.deliveries.size
 
+    click_button("Select recipients")
+    find('a', :text => "Incomplete").click
+    page.should have_field('To', with: 'gial.ackbar@rebel-alliance.org,luke.skywalker@rebel-alliance.org,evram.lajaie@rebel-alliance.org,c3po@droids.com')
```

Comment:
Let's shorten this test by deirectly going to the email view of that event. And we can also remove the selection of recipients, since we just want to test canceling the mail here.